TwitchDownloaderWPF/MainWindow.xaml.cs

```diff
@@ -84,6 +84,14 @@ private async void Window_Loaded(object sender, RoutedEventArgs e)
 #endif
         }
 
+        internal static string[] GetTemplateSubfolders(ref string fullPath)
+        {
+            string[] returnString = fullPath.Split(new char[] { '\\', '\\', '/'}, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Is there a reason `\` is part of the char[] twice?

Reply:
No, I've accidentally duplicated it.